modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -763,6 +773,24 @@ void ONNXImporter::populateNet(Net dstNet)
             layerParams.set("end_axis", axis);
             layerParams.type = "Flatten";
         }
+        else if (layer_type == "Flatten")
+        {
+            CV_Assert_N(node_proto.input_size() == 1);
```

Comment:
> _N

Why?

---

Please use `CV_CheckEQ(node_proto.input_size(), 1, "");` macro instead of assertion for better diagnostic message.